awesome_provider/app/src/routing_handler.py

```diff
@@ -5,7 +5,7 @@ def validate_time_format(time_to_validate):
     date_format = '%Y%m%d%H%M%S'
     try:
         date_obj = datetime.strptime(time_to_validate, date_format)
-        print(date_obj)
-        return True, "Succeeded"
-    except ValueError:
-        return False, "Incorrect data format, should be YYYYMMDDHHMMSS"
+        return True
+    except ValueError as e:
+        raise ValueError(f"Incorrect data format, {date_format} " \
+              f"should be YYYYMMDDHHMMSS \b {e}")
```

Comment:
Revert these changes, although this is the right way. For demo purposes we have to continue .